prescription/integrity.go

```diff
@@ -58,7 +47,7 @@ func generateJCSSha512(attributes IntegrityAttributes) (string, error) {
 	return fmt.Sprintf("%x", hash), nil
 }
 
-func MustGenerateIntegrityHash(attributes IntegrityAttributes) IntegrityHash {
+func MustGenerateIntegrityHash(attributes DataAttributes) IntegrityHash {
 	hash, err := generateJCSSha512(attributes)
 	if err != nil {
 		panic(err)
```

Comment:
There's not really much we can do if we can't generate the has. Why bubble up the error? Panicing in `Must...` functions is a fairly standard pattern

Reply:
I agree that a panic in a Must function is common, but crashing the service is not the right call. If there were literally nothing else that the program could do, sure, but presumably the service could continue to function in a read-only capacity until the issue is fixed. [^1]

[^1]: If the problem can be masked or worked around, it's always better to let things continue to run rather than taking down the whole program. One possible counterexample is during initialization: if the library truly cannot set itself up, it might be reasonable to panic, so to speak. -- https://go.dev/doc/effective_go#panic

Reply:
Panics are handled by the server and a 500 error is returned to the client - we're not crashing the program